### PR TITLE
 track attestations sent/received in grafana

### PR DESCRIPTION
**Pull request by tersec**
2019-10-25-170329_853x429_scrot

It's not clear what the best layout will be eventually, but for now, this targets specifically what seems to be one of the more bottlenecking issues.

### CODE REVIEW

**Comment by stefantalpalaru** [comment]
If you add "#0" to all panel titles, I can convert it to the specific node number: https://github.com/status-im/nim-beacon-chain/blob/a9fcaf16a65513e94d082fddb8f1b4ef62e0977e/tests/simulation/process_dashboard.nim#L43

**Comment by tersec** [comment]
> If you add "#0" to all panel titles, I can convert it to the specific node number:
> 
> https://github.com/status-im/nim-beacon-chain/blob/a9fcaf16a65513e94d082fddb8f1b4ef62e0977e/tests/simulation/process_dashboard.nim#L43

The difference is deliberate -- sent is across all nodes. That's why they match. Does it make sense when all the sent attestation counts are going to be identical?

File: tests/simulation/beacon-chain-sim-node0-Grafana-dashboard.json [hunk 1]
```diff
@@ -408,6 +408,174 @@
...
+        "full": false,
+        "lineColor": "rgb(31, 120, 193)",
+        "show": false,
+        "ymax": null,
+        "ymin": null
+      },
+      "tableColumn": "",
+      "targets": [
+        {
+          "expr": "sum(beacon_attestations_sent_total)",
```
Comment (stefantalpalaru) [inline tests/simulation/beacon-chain-sim-node0-Grafana-dashboard.json]: I think this needs to be the counter's current value, not the sum of all its values over time.

Comment (tersec) [reply tests/simulation/beacon-chain-sim-node0-Grafana-dashboard.json]: That appears to function as the current values across all nodes. If that's not the documented way of doing so, happy to use the correct approach.

**Comment by tersec** [comment]
> Oh, OK. So `sum()` works only over values with different "node" attributes, not different timestamps?

For me, yes. I haven't doublechecked the Prometheus documentation or the data model in question.

**Comment by stefantalpalaru** [comment]
Worth adding something like "(all nodes)" to the panel title, so we don't forget.

**Comment by stefantalpalaru** [comment]
Yes, Prometheus aggregation functions like `sum()` work over labels ("dimensions"): https://prometheus.io/docs/prometheus/latest/querying/operators/#aggregation-operators

**Pull request merged**